apps/judicial-system/web/src/routes/PublicProsecutor/Tables/CasesReviewed.tsx

```diff
@@ -0,0 +1,107 @@
+import React from 'react'
+import { useIntl } from 'react-intl'
+import { AnimatePresence } from 'framer-motion'
+
+import { Tag, Text } from '@island.is/island-ui/core'
+import { capitalize, formatDate } from '@island.is/judicial-system/formatters'
+import { core, tables } from '@island.is/judicial-system-web/messages'
+import { SectionHeading } from '@island.is/judicial-system-web/src/components'
+import { useContextMenu } from '@island.is/judicial-system-web/src/components/ContextMenu/ContextMenu'
+import {
+  CourtCaseNumber,
+  DefendantInfo,
+} from '@island.is/judicial-system-web/src/components/Table'
+import Table, {
+  TableWrapper,
+} from '@island.is/judicial-system-web/src/components/Table/Table'
+import TableInfoContainer from '@island.is/judicial-system-web/src/components/Table/TableInfoContainer/TableInfoContainer'
+import {
+  CaseListEntry,
+  IndictmentCaseReviewDecision,
+} from '@island.is/judicial-system-web/src/graphql/schema'
+
+import { strings } from './CasesReviewed.strings'
+
+interface Props {
+  loading: boolean
+  cases: CaseListEntry[]
+}
+
+const CasesReviewed: React.FC<Props> = ({ loading, cases }) => {
```

Comment:
I've been thinking that we should start importing FC from react instead of writing `React.FC`. We import everything else from react, useMemo, useCallback, etc. so it's more clearer if we import FC as well (IMO :D)

Reply:
That's a good point, I agree! I changed it here, we should do another separate PR for cleanup then for the rest as there's about 165 other places we need to update to make this the new rule 😅 A good cooldown task